.github/workflows/doc-tests.yml

```diff
@@ -17,14 +17,37 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        builder: [ html ]
+        builder: [ html, linkcheck ]
         include:
           # Run default html builder with warnings as error
           - builder: html
             args: -W
     steps:
       - uses: actions/checkout@v4
+      - name: Show me your git
```

Comment:
@LecrisUT  what is this good for exactly?

Reply:
Sorry, these are from a bunch of debug commits that I've added to debug: https://github.com/teemtee/tmt/pull/2489#issuecomment-2158802072 i.e. how to get the links to point to PR files. Will remove those, but need some feedback on:
```console
$ git for-each-ref --format="%(refname)" --points-at HEAD
```

See the github action log on what everything is expanded to. For PRs `refs/remotes/pull/2489/merge` works well (replace that in the commitish) and anything that can expand to that is a good approach (needs to expand to a `ref` that is recognized on the target repo)